docs/code/reference.md

```diff
@@ -273,13 +274,13 @@ elementary has always been known for its strong focus on great design, but if yo
 
 Let’s say you spent the last few hours re-designing the Search Engine Manager dialog in Midori and you want to bring this work to the attention of Midori’s developers. You could file a bug report something like “Search engine popup sucks” and paste your mockup and be done. But that approach isn’t going to win you any friends and your report will probably be marked “opinion” or “invalid”. Instead, we should consider the amount of work it will take to implement the new design and try to present it in a way that gets developers excited about the overall vision without demanding 1,000 lines of code in one shot.
 
-## Use Blueprints {#use-blueprints}
+## Use GitHub {#use-blueprints}
```

Comment:
Seeing that almost everything happens on Github this title is a bit broad?
Maybe something like "Open a Github issue"? I'm not sure here.